src/library/features/crates/cratefeature.cpp

```diff
@@ -463,7 +456,7 @@ void CrateFeature::slotDeleteCrate() {
             if (QMessageBox::question(
                   nullptr,
                   tr("Deleting multiple crates"),
-                  tr("Deleting this crate will also delete all it's childer. Are you sure?"),
+                  tr("Deleting this crate will also delete all it's children. Are you sure?"),
```

Comment:
Use "its" for the possessive pronoun, not "it's". "Are you sure?" is a little ambiguous. Expand it to, "Are you sure you want to delete this crate?"